sections/title.py

```diff
@@ -153,29 +137,26 @@ def title_section():
                                         "Effective Altruism",
                                         href="https://www.effectivealtruism.org/"
                                     ),
-                                    ' is a loose collective of quantitatively-minded philanthropists and do-gooders.',
+                                    ' (EA) is a movement that uses reason and evidence to do the most good.',
                                 ]
                             ),
                             html.P(
-                                'There are several EA organisations which collect data on grants, donors, and pledges.',
-                            ),
-                            html.P(
-                                'This website aggregates and visualizes that data.',
+                                'This website aggregates and visualizes data from several EA organisations!',
                             ),
                             contents(),
                             html.P(
                                 [
-                                    'Source code is available on ',
+                                    'All of the source code is available on ',
```

Comment:
Leave original (shorter)

Reply:
What about the 'There are several EA organisations which collect data on grants, donors, and pledges.' part?